src/graphql/graphql.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@nestjs/common'
+import { GraphQLClient, gql } from 'graphql-request'
+import { Logger } from '@nestjs/common'
+
+const API_BASE = 'https://bpi.civicengine.com/graphql'
+const BALLOT_READY_KEY = process.env.BALLOT_READY_KEY
+
+@Injectable()
+export class GraphqlService {
+  private readonly logger = new Logger(GraphqlService.name)
+  constructor(private readonly graphQLClient: GraphQLClient) {}
+
+  async fetchGraphql(query: any, variables?: any) {
+    let data: any
+
+    const graphQLClient = new GraphQLClient(API_BASE, {
+      headers: {
+        authorization: `Bearer ${BALLOT_READY_KEY}`,
+        'Content-Type': 'application/json',
+      },
+    })
```

Comment:
This is still being created on _every_ call to this method.

This should be a `private` member on this class as mentioned in the last PR here: https://github.com/thegoodparty/gp-api/pull/33#discussion_r1890938951